oak_attestation/src/dice.rs

```diff
@@ -180,113 +170,99 @@ impl TryFrom<DiceData> for DiceBuilder {
             .eca_private_key;
         let signing_key = SigningKey::from_slice(eca_private_key).map_err(anyhow::Error::msg)?;
 
+        // Zero out the ECA private key if it was set.
+        if let Some(certificate_authority) = &mut value.certificate_authority {
```

Comment:
Can we just zero out the entire DiceData instead? Perhaps by wrapping it into a Zeroizing wrapper at the beginning of this method: https://docs.rs/zeroize/latest/zeroize/#zeroizingz-wrapper-for-zeroizing-arbitrary-values-on-drop

Reply:
Not sure it could be implemented for `DiceData`:
```
error[E0277]: the trait bound `DiceData: DefaultIsZeroes` is not satisfied
   --> oak_attestation/src/dice.rs:162:44
    |
162 |         let mut dice_data = Zeroizing::new(value);
    |                             -------------- ^^^^^ the trait `DefaultIsZeroes` is not implemented for `DiceData`
    |                             |
    |                             required by a bound introduced by this call

```